eslint.config.mjs

```
@@ -63,22 +69,69 @@ export default [
+      // honours the `^__` ignore pattern (see its options above); the
+      // sonarjs version has no configurable options so it can't
+      // exempt intentionally-discarded destructuring targets like
+      // `const { result: __result, ...rest } = ...`. Disable to avoid
+      // double-reporting and to let the `__` convention work.
+      "sonarjs/no-unused-vars": "off",
       // MulmoClaude is a local desktop app — spawning claude/docker/git
       // via PATH is normal operation, not a server-side injection risk.
       "sonarjs/no-os-command-from-path": "off",
-      "sonarjs/cors": "off",
-      "sonarjs/pseudo-random": "warn",
+      "sonarjs/cors": "off"
```

ESLint

[error] Insert `,`

(prettier/prettier)